pkg/controller/cassandradatacenter/controller/operator.go

```diff
@@ -0,0 +1,21 @@
+package controller
+
+import (
+	cassandraoperatorv1alpha1 "github.com/instaclustr/cassandra-operator/pkg/apis/cassandraoperator/v1alpha1"
+	"k8s.io/api/apps/v1beta2"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type CassandraOperator interface {
+	CreateOrUpdateNodesService(cdc *cassandraoperatorv1alpha1.CassandraDataCenter) (*v1.Service, error)
```

Comment:
Why do these need to be interface functions? It looks like their receiver method implementations just call to the global functions...